Source/WebCore/css/CSSToStyleMap.cpp

```diff
@@ -634,4 +634,18 @@ void CSSToStyleMap::mapNinePieceImageRepeat(const CSSValue& value, NinePieceImag
     image.setVerticalRule(fromCSSValue<NinePieceImageRule>(value.second()));
 }
 
+void CSSToStyleMap::mapAnimationRangeStart(Animation& animation, const CSSValue& value) const
```

Comment:
can you move the new methods with the other animation methods?

Reply:
Fixed.